### ISSUE TITLE
Create inflated and deflated defense area functions

### ISSUE THREAD

**Issue by jonathanlew** [UBC-Thunderbots/Software#1332]
### Description of the task
Create inflated and deflated defense area functions to replace some uses of Rectangle::expand. 
When we expand/shrink a defense area using Rectangle::expand, we're also moving the goal line since expand is w/r to the centre point, which can cause unexpected behaviour. Another problem is that different places inflate the defense area different amounts, when it can probably be standardized. 

This issue is about creating these common functions and integrating them. This issue might improve how the goalie works.

At this point, Rectangle::expand may not be used as it was originally intended, so please open a new issue on how to clean that function up if required

### Acceptance criteria

- [ ] Create inflated and deflated defense area functions 
- [ ] Integrate them where appropriate
- [ ] New ticket to clean up Rectangle::expand if required

### Blocked By
we should see how #1331 plays out first
